yarn-project/validator-client/src/validator.ts

```diff
@@ -289,9 +288,27 @@ export class ValidatorClient extends (EventEmitter as new () => WatcherEmitter)
 
     if (!validationResult.isValid) {
       this.log.warn(`Proposal validation failed: ${validationResult.reason}`, proposalInfo);
-      incFailedAttestation(validationResult.reason || 'unknown');
 
-      // Slash invalid block proposals
+      // Only track attestation failure metrics if we're actually in the committee
+      if (partOfCommittee) {
```

Comment:
Heads up this means we get no metrics on failed proposals even when setting `alwaysReexecuteBlockProposals`.  Not sure this is what we want.